### ISSUE TITLE
UnicodeDecodeError when pip install

### ISSUE THREAD

**Issue by jmrobles** [divio/aldryn-apphooks-config#71]
When I try to install via pip (inside docker image), setuptools fails with the following UnicodeDecode error:
```
   Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-5g_7sa22/aldryn-apphooks-config/setup.py", line 15, in <module>
        long_description=open('README.rst').read(),
      File "/usr/lib/python3.5/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 1293: ordinal not in range(128)
```

**Comment by NeckBeardPrince** [comment]
Same issue. Was this ever resolved? looks like there has been no movement on this issue.

**Comment by yakky** [comment]
Fixed in #74 which removed the table completely
